USAGE_android.md

```diff
@@ -260,6 +514,215 @@ originally named `gfxrecon_capture.gfxr` and was created at 2:35 PM
 on November 25, 2018:
   `gfxrecon_capture_20181125T143527.gfxr`
 
+
+#### Pulling Down a Trace from the Device
+
+First check the directory to make sure a trace exists:
+
+```bash
+adb shell ls ${location}
+```
+
+For example:
+
+```bash
+adb shell ls /sdcard
+```
+
+If the trace exists there, for example as "gfxrecon_trace.gfxr", it can be
+extracted off of the Android device using the `adb pull` command:
+
+```bash
+adb pull ${location}/${filename}
+```
+
+Such as:
+
+```bash
+adb pull /sdcard/gfxrecon_trace.gfxr
+```
+
+This will pull the file down to your host machine's current directory.
+
+### Limitations of Capture On Android
+
+The current version of the Android capture layer has the following limitations:
+
+ * The capture layer must be deployed and enabled manually.
+   Future work may provide a tool to automate the process of launching a
+   debuggable app, injecting the trace layer with the Java Debug Wire Protocol
+   (JDWP), and enabling the layer.
+ * Writing files to external storage requires the external storage permission.
+   If the app to be captured does not already have this permission, it can be
+   granted manually through the Applications panel from device Settings.
+ * Adding a timestamp to the filename does not guarantee a unique filename.
+   Some applications will query for Vulkan support by creating an instance,
+   retrieving instance properties, and then destroying the instance.
+   After the initial support query, the application will create a second
+   instance to do its real work.
+   Each of these Vulkan sessions will generate a capture file.
+   If the time elapsed between these two sessions is less than a second, both
+   sessions will use the same file name, with the second session overwriting the
+   file produced by the first session.  Future improvements to the capture layer
```

Comment:
I don't think we need to talk about "future improvements" but if you think it's really important, I think this might read better as "Future improvements to the capture layer may include resolving this issue."

Reply:
No.  This is a section I pulled from somewhere else.  I'll just strip it.